lib/active_fulfillment/services/amazon_mws.rb

```diff
@@ -53,18 +54,20 @@ class AmazonMarketplaceWebService < Service
       :list_inventory => {
         :sku => "SellerSkus.member.%d"
       }
-    }
+    }.freeze
+
+    SHIPPING_METHODS = [
+      [ 'Standard Shipping', 'Standard' ],
+      [ 'Expedited Shipping', 'Expedited' ],
+      [ 'Priority Shipping', 'Priority' ]
+    ].freeze
 
     # The first is the label, and the last is the code
     # Standard:  3-5 business days
     # Expedited: 2 business days
     # Priority:  1 business day
     def self.shipping_methods
-      [
-        [ 'Standard Shipping', 'Standard' ],
-        [ 'Expedited Shipping', 'Expedited' ],
-        [ 'Priority Shipping', 'Priority' ]
-      ].inject({}){|h, (k,v)| h[k] = v; h}
+      SHIPPING_METHODS.inject({}){|h, (k,v)| h[k] = v; h}
```

Comment:
I realize the old code was like this but would it be possible to just define this as a hash in the first place?